crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -359,11 +359,6 @@ impl CumulocityConverter {
                 let ancestors_external_ids =
                     self.entity_store.ancestors_external_ids(entity_topic_id)?;
 
-                if self.config.bridge_in_mapper && entity_topic_id.is_bridge_health_topic() {
-                    // Skip service creation for the mapper-inbuilt bridge, as it is part of the mapper service itself
-                    return Ok(vec![]);
-                }
-
```

Comment:
Replaced this with a better logic higher up the stack on the auto-registration path, which was the only path where this logic was relevant.